test/controllers/v1/EnterpriseUnitRoutingSpec.scala

```diff
@@ -0,0 +1,106 @@
+package controllers.v1
+
+import scala.concurrent.Future
+
+import play.api.inject.guice.GuiceApplicationBuilder
+import play.api.mvc.Result
+import play.api.test.FakeRequest
+import play.api.test.Helpers._
+import org.scalatest.{ FreeSpec, Matchers }
+import org.scalatestplus.play.guice.GuiceOneAppPerSuite
+
+import controllers.v1.fixture.HttpServerErrorStatusCode
+
+class EnterpriseUnitRoutingSpec extends FreeSpec with GuiceOneAppPerSuite with Matchers {
+
+  override def fakeApplication() = new GuiceApplicationBuilder().configure(Map("db.hbase-rest.timeout" -> "100")).build()
+
+  private trait Fixture extends HttpServerErrorStatusCode {
+    val ValidErn = "1000000012"
+    val ValidPeriod = "201803"
+
+    val fakeRequest: (String) => Option[Future[Result]] = (url: String) => route(app, FakeRequest(GET, url))
+  }
+
+  "A request to retrieve a Enterprise Unit by an Enterprise reference number (ERN) and period (yyyyMM)" - {
+    "is rejected when" - {
+
+      "the ERN is" - {
+        "has fewer than 10 digits" in new Fixture {
```

Comment:
Not a big deal - as this is very subtle.
Tests follow the 3 A's pattern: Arrange, Act, Assert.
It is helpful to distinguish these 3 phases within the test.  I do this by leaving a blank line between each phase.  Others separate each phase with comments, such as: 
// Given
// When
// Then 

So while removing the blank line seems trivial, it has collapsed the Arrange & Act phases together.  I told you this was subtle ...